pkgs/applications/networking/instant-messengers/telegram/kotatogram-desktop/default.nix

```diff
@@ -23,16 +23,20 @@ mkDerivation rec {
   buildInputs = [
     qtbase qtimageformats ffmpeg_3 openalSoft lzma lz4 xxHash libdbusmenu
     zlib minizip openssl hunspell libtgvoip microsoft_gsl tl-expected range-v3
+    libopus alsaLib libpulseaudio
   ];
 
   qtWrapperArgs = [
     "--prefix PATH : ${xdg-utils}/bin"
   ];
 
   cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE=Release"
     "-DTDESKTOP_API_TEST=ON"
+    "-DDESKTOP_APP_USE_PACKAGED_GSL=OFF"
     "-DDESKTOP_APP_USE_PACKAGED_RLOTTIE=OFF"
     "-DDESKTOP_APP_USE_PACKAGED_VARIANT=OFF"
+    "-DTDESKTOP_USE_PACKAGED_TGVOIP=OFF"
```

Comment:
Took it from here: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=kotatogram-desktop#n60